virtcontainers/factory/template/template.go

```diff
@@ -171,3 +178,10 @@ func (t *template) checkTemplateVM() error {
 	_, err = os.Stat(t.statePath + "/state")
 	return err
 }
+
+// Logger returns a logrus logger appropriate for logging template messages
+func (t *template) Logger() *logrus.Entry {
```

Comment:
Thanks for adding this. But can you "stitch" this into the existing logging facility by creating a `SetLogger()` function here and then calling it in:

- https://github.com/kata-containers/runtime/blob/master/virtcontainers/api.go#L49

Then, Having this logger function just return the global logger in this module that will now contain all the log fields. This is required for consistency and for https://github.com/kata-containers/tests/tree/master/cmd/log-parser#logfile-requirements.

Reply:
I have add SetLogger, ptal. thanks @jodh-intel